src/libs/auth.test.js

```diff
@@ -0,0 +1,144 @@
+import { parseToSDetails } from 'src/libs/auth'
+
+
+const tosDetailsDefault = {
+  currentVersion: '1',
+  isEnabled: true,
+  isGracePeriodEnabled: true,
+  userAcceptedVersion: '1'
+}
+
+const parsedTosDetailsDefault = {
+  isGracePeriodEnabled: true,
+  currentVersion: '1',
+  userAcceptedVersion: '1',
+  userCanUseTerra: true,
+  showTosPopup: false
+}
+
+describe('Terms of Service Detail Parser gives access', () => {
+  it('gives access to users that have accepted the most recent ToS version', () => {
+    expect(parseToSDetails(tosDetailsDefault)).toEqual(parsedTosDetailsDefault)
```

Comment:
lets break these up to follow AAA pattern with marked sections:
// Arrange
(setup and get to a precondition if needed)
// Act
(do the main thing/action being tested)
const result = doTheThing()
// Assert
expect(result).toEqual(...}

Reply:
Understood, was going to do this, but given that this is a prod issue and these are the first tests, we just wanted to get this in.  And it's a Friday and we're trying to get this out so that people have a chance to look at it before we release.  You know me and know I care a lot about having good unit tests, so it's definitely a trade off.  Are you OK if we leave the tests as is for now so we can just get this merged ASAP?

Reply:
that works :)